README.md

```
@@ -65,136 +57,311 @@ counters. Follow this
 ```
 
-> **_NOTE_**: if you do not provide `triton_path` the triton repo will be cloned
-at container startup time.
+---
 
-> **_NOTE_**: if you do provide a triton_path you should run `git submodule init`
-and `git submodule update` on the mounted repo if you haven't already run these
-commands.
+### Running
+
+#### Options
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)